app/src/components/buttons/index.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import { FlipTokensButton } from './flip-tokens-button'
+import { IconButton } from './icon-button'
+import { SelectTokenButton } from './select-token-button'
+import { StandardButton } from './standard-button'
+import { PresetButton } from './preset-button'
+
+export {
+  FlipTokensButton,
+  IconButton,
+  SelectTokenButton,
+  StandardButton,
+  PresetButton
+}
```

Comment:
@josheleonard to confirm, but I think we don't need to maintain these `index.ts` files anymore.

Reply:
We will need this file structure if we plan on making this a component library no?
Else we will have multiple imports to the `buttons` directory everywhere, etc..

Reply:
The repo name and current structure suggests that this project is a self-contained app containing the entire Swap experience (which could be loaded via iframe). If we are creating a library, are we exposing individual UI elements, or a single, self-contained Swap component?

These barrel files only make sense if we are exposing individual components for building a swap UI. Even then, a root-level index.ts should be the only file handling exposing our chosen library exports

Reply:
See: https://vitejs.dev/guide/build.html#library-mode for setting up the build step to output a library

Reply:
Let's discuss this further and make a separate issue to address restructure if needed. 🙂